package.json

```diff
@@ -17,7 +17,7 @@
         "move-back": "mv tmp/README.md svg/README.md && mv tmp/package.json svg/package.json && rm -rf tmp",
         "copy": "yarn move && rm -r svg && mkdir svg && copyfiles -f -e \"design/old/*.svg\" -e \"design/Internal/*.svg\" -e \"design/WIP/*.svg\" -e \"design/Products/**/*.svg\" \"design/Products/*.svg\" design/**/*.svg design/**/**/*.svg svg && yarn move-back",
         "sprites": "create-svg-sprite --optimize --input './svg' --output './svg' --name 'icons'",
-        "build": "yarn build:font",
+        "build": "yarn build:font && yarn build:rn-vector",
```

Comment:
This build command is meant to be just for the icons font and svgs. We've been keeping the others separate. If you add the build for rn vector icons here, 1) it may not have its dependencies installed yet and 2) CircleCI will be building the rn vector icons twice.

Reply:
I went ahead and changed this back for you